src/app/MessageDef/ReadRequestMessage.h

```diff
@@ -71,42 +64,40 @@ class Parser : public StructParser
     /**
      *  @brief Get a TLVReader for the AttributePathIBs. Next() must be called before accessing them.
      *
-     *  @param [in] apAttributePathList    A pointer to an attribute path list parser.
+     *  @param [in] apAttributeRequests    A pointer to an attribute path list parser.
```

Comment:
Need to update the comment which no more point to attribute path

Reply:
it is still an attribute path list parser..although spec name this member as attribute request